EIPS/eip-5380.md

```diff
@@ -0,0 +1,181 @@
+---
+eip: 5380
+title: EIP-4907 Alternative Design
+description: Add a time-limited role with restricted permissions to EIP-721 tokens.
+author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>, Tim Daubenschütz (@TimDaub)
```

Comment:
@Pandapip1 Since you are the one authoring this, I recommend adding yourself as an author, otherwise you'll need to wait to merge this as draft (and any subsequent steps in the process) until an author approves.

Reply:
I thought I had done that.
```suggestion
author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>, Tim Daubenschütz (@TimDaub), Pandapip1 (@Pandapip1)
```